src/features/annotation-thread/__tests__/AnnotationThreadContent.test.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+
+import { mount } from 'enzyme';
+import React from 'react';
+import { AnnotationThreadContentComponent } from '../AnnotationThreadContent';
+import { annotation } from '../../../__mocks__/annotations';
+import ActivityThread from '../../../elements/content-sidebar/activity-feed/activity-feed/ActivityThread';
+import AnnotationActivity from '../../../elements/content-sidebar/activity-feed/annotations';
+
+describe('elements/annotation-thread/AnnotationThreadContent', () => {
```

Comment:
Should this path be updated or removed? This test suite also seems quite light given the amount of logic in AnnotationThreadContent.